CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # Changelog
 
+## [1.28.0] - 2024-10-07
+
+## What's Changed
```

Comment:
For now it's just hand-picked lines from the GitHub-generated changelog...

Reply:
GitHub Copilot gives an interesting summary for this log. Might be an acceptable starting point...

<details><summary>Summary</summary>

The summary of the changelog in PR#4041 is categorized as follows:

### New Features
- Make `reqwest/rustls` the new default [pr#3798](https://github.com/rust-lang/rustup/pull/3798)
- Reflect the download/TLS backends in the user agent [pr#3815](https://github.com/rust-lang/rustup/pull/3815)
- Add `--quiet` to `rustup toolchain list` and friends [pr#3862](https://github.com/rust-lang/rustup/pull/3862)
- Use `rustls-platform-verifier` [pr#3903](https://github.com/rust-lang/rustup/pull/3903)
- Add hint to run `rustup self` when err desc is `self` [pr#3901](https://github.com/rust-lang/rustup/pull/3901)
- Set log level to `WARN` on `-q` if `RUSTUP_LOG` is unset [pr#3911](https://github.com/rust-lang/rustup/pull/3911)
- Add loongarch64-unknown-linux-musl support [pr#3921](https://github.com/rust-lang/rustup/pull/3921)
- Warn when removing the default/active toolchain [pr#3924](https://github.com/rust-lang/rustup/pull/3924)
- Refine suggestions regarding manifest checksum mismatches [pr#3923](https://github.com/rust-lang/rustup/pull/3923)
- Install the active toolchain by default on `rustup toolchain install` [pr#3983](https://github.com/rust-lang/rustup/pull/3983)
- Remove implicit toolchain installation [pr#3985](https://github.com/rust-lang/rustup/pull/3985)
- Use `aws-lc` instead of `ring` [pr#3898](https://github.com/rust-lang/rustup/pull/3898)
- Set log level to `INFO`/`DEBUG` on `--quiet`/`--verbose` if `RUSTUP_LOG` is unset [pr#3987](https://github.com/rust-lang/rustup/pull/3987)

### Bug Fixes
- Work around `hyper` hang issue by adjusting `reqwest` config [pr#3855](https://github.com/rust-lang/rustup/pull/3855)
- Fix misleading "uninstalled toolchain" notification [pr#3869](https://github.com/rust-lang/rustup/pull/3869)
- Handle the possible unavailability of `/proc` in `rustup-init.sh` [pr#3800](https://github.com/rust-lang/rustup/pull/3800)
- Don't install toolchain on `rustup --version` [pr#3948](https://github.com/rust-lang/rustup/pull/3948)
- Return `ExitCode(1)` when `update()` fails [pr#3952](https://github.com/rust-lang/rustup/pull/3952)
- Warn if host is incompatible with the toolchain in `rustup default` [pr#3980](https://github.com/rust-lang/rustup/pull/3980)
- Consider possible renames in `Component::try_new()` [pr#3991](https://github.com/rust-lang/rustup/pull/3991)

### Refactors
- Rewrite `rustup-init` with `clap_derive` [pr#3814](https://github.com/rust-lang/rustup/pull/3814)
- Rewrite `rustup` with `clap_derive` [pr#3596](https://github.com/rust-lang/rustup/pull/3596)
- Refactor around use of `Process` [pr#3860](https://github.com/rust-lang/rustup/pull/3860)
- Pass Process around explicitly [pr#3871](https://github.com/rust-lang/rustup/pull/3871)
- Deny installing a host-incompatible toolchain w/o `--force-non-host` [pr#4028](https://github.com/rust-lang/rustup/pull/4028)
- Replace `winreg` with `windows-registry` [pr#3896](https://github.com/rust-lang/rustup/pull/3896)

### Windows-Specific Improvements
- Replace remaining `winapi` usage with `windows-sys` [pr#3802](https://github.com/rust-lang/rustup/pull/3802)
- Add Windows Arm64 to Other Installation Methods [pr#3854](https://github.com/rust-lang/rustup/pull/3854)
- Finalise Rustup Windows Arm64 support [pr#3840](https://github.com/rust-lang/rustup/pull/3840)

### Miscellaneous
- Reduce code duplication for show commands [pr#3813](https://github.com/rust-lang/rustup/pull/3813)
- Make `rustup show` output info in a more logical order [pr#3225](https://github.com/rust-lang/rustup/pull/3225)
- Stop showing ETA after download is complete [pr#3827](https://github.com/rust-lang/rustup/pull/3827)
- Switch from sync to a tokio runtime [pr#3367](https://github.com/rust-lang/rustup/pull/3367)
- Consistently add context with file path when parsing fails [pr#3853](https://github.com/rust-lang/rustup/pull/3853)
- Simplify process interface [pr#3764](https://github.com/rust-lang/rustup/pull/3764)
- Use serde to encode/decode various TOML formats [pr#3864](https://github.com/rust-lang/rustup/pull/3864)
- Ship `tracing` and friends by default [pr#3803](https://github.com/rust-lang/rustup/pull/3803)
- Inline errors, rename TLS backend [pr#3888](https://github.com/rust-lang/rustup/pull/3888)
- Mention `RUSTUP_LOG` and console-based tracing in dev guide [pr#3879](https://github.com/rust-lang/rustup/pull/3879)
- Allow `rustup doc` to search for unions [pr#4004](https://github.com/rust-lang/rustup/pull/4004)
- Try symlinking proxies first, falling back to hardlinking if that fails [pr#4023](https://github.com/rust-lang/rustup/pull/4023)

You can view more details [here](https://github.com/rust-lang/rustup/pull/4041/files).

</details>